sites-enabled/10-www.freecodecamp.org.conf

```diff
@@ -186,6 +200,20 @@ server {
   }
 }
 
+server {
+  listen 443 ssl http2;
+  listen [::]:443 ssl http2;
+
+  server_name forum.freecodecamp.org;
+
+  # SSL
+  include snippets/common/ssl-freecodecamp-org.conf;
+  
+  location /c/italian {
+    return 302 https://forum.freecodecamp.org;
+  }
+}
+
```

Comment:
```suggestion

```

I missed that we can't map the forum DNS over to our NGINX instances. It's likely that we need to add something within the forum itself.

Reply:
Ah, right. We could change the client logic, too.